pmlc/rt/vulkan/vulkan_invocation.cc

```diff
@@ -565,13 +565,22 @@ void VulkanInvocation::createPipelineLayout() {
                      "vkCreatePipelineLayout");
 }
 
-void VulkanInvocation::createComputePipeline() {
+void VulkanInvocation::createComputePipeline(uint32_t subgroupSize) {
   if (!curr) {
     throw std::runtime_error{"createComputePipeline: curr is nullptr!"};
   }
 
+  VkPipelineShaderStageRequiredSubgroupSizeCreateInfoEXT subgroupSizeInfo;
+  subgroupSizeInfo.sType = VK_STRUCTURE_TYPE_PIPELINE_SHADER_STAGE_CREATE_INFO;
```

Comment:
Changed intentionally from VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_SUBGROUP_PROPERTIES to VK_STRUCTURE_TYPE_PIPELINE_SHADER_STAGE_CREATE_INFO based on debug results from @jbruestle